### PR TITLE
RHD common Drupal code standards

### PR DESCRIPTION
**Pull request by jordanpagewhite**
### Description

This applies some Drupal coding standards to the rhd_common module PHP files. There are also a few changes that are not strictly within the Drupal coding standards but I feel worthwhile to improve readability.

I'd be happy to discuss any change with anyone. Let me know what you think. Thanks.

### JIRA Issue Link
* n/a

### Verification Process

* Everything should function exactly as it did prior to these commits
* Aside from typehinting and maybe 1-2 exceptions, the files in the rhd_common module should conform to the Drupal coding standards

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/modules/custom/rhd_common/rhd_common.sitemap.inc [hunk 1]
```diff
@@ -75,13 +84,18 @@ function build_subpage_link_entry($sub_page, $template)
 }
 
 /**
- * Returns an array af links which are for product nodes
- * @param array $links Links in the sitemap
- * @param array $product_ids node ids of products
- * @return array An array with the link entry for a product, the key is key in the $links array
+ * Returns an array af links which are for product nodes.
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/modules/custom/rhd_common/rhd_common.sitemap.inc]: ... an array _of_ links ...

Comment (jordanpagewhite) [reply _docker/drupal/drupal-filesystem/web/modules/custom/rhd_common/rhd_common.sitemap.inc]: Thanks for catching that.

I also agree that some of the coding standard changes are fairly worthless and pedantic. I am just trying to follow some coding standard, but it doesn't particularly matter to me that it be the Drupal coding standards. If you have other preferences, let me know. I don't want to step on your toes in any way. As long as we are improving readability and adding helpful comments/documentation as much as is reasonable, then I am a very happy camper.

**Comment by robpblake** [comment]
`DEBUG - executing command /var/www/drupal/vendor/bin/drush ["--root=/var/www/drupal/web", "cr"]
drupal_install_checker.rb:10:in `exec!': PHP Fatal error:  Cannot redeclare rhd_common_form_alter() (previously declared in /var/www/drupal/web/modules/custom/rhd_common/rhd_common.module:23) in /var/www/drupal/web/modules/custom/rhd_common/rhd_common.form_validations.inc on line 13 (RuntimeError)
 [error]  Drush command terminated abnormally due to an unrecoverable error.
Error: Cannot redeclare rhd_common_form_alter() (previously declared in /var/www/drupal/web/modules/custom/rhd_common/rhd_common.module:23) in /var/www/drupal/web/modules/custom/rhd_common/rhd_common.form_validations.inc, line 13 
	from drupal_install_checker.rb:106:in `drush_cache_rebuild'
	from drupal_install_checker.rb:29:in `prepare_drupal_for_boot'
	from drupal_install_checker.rb:133:in `<main>'
DEBUG - execution of command '/var/www/drupal/vendor/bin/drush ["--root=/var/www/drupal/web", "cr"]' has failed with status code='255'`

@jordanpagewhite It's failing because of the above error

**Comment by jordanpagewhite** [comment]
I resolved that redeclaration of hook_form_alter() on Friday on my local machine and committed it, but I 
 forgot to push it up to the remote. Thanks Rob for keeping an eye on this. I was pretty confused for a moment why my local branch wasn't matching what I saw here, but `git log` saved the day again.

**Pull request merged**